### PR TITLE
Roll 34 global app font & navigation tabs

### PR DESCRIPTION
**Pull request by brandlxee11**
Font throughout the app has been changed and the dashboard of the app has been created.

Coming up next...

Login page flow w/ sign in & out (front-end only)
Card editor page

### CODE REVIEW

**Comment by brickgriff** [comment]
Hmm... I see this is called `ROLL-34` but the branch is `roll-33`. Please send ROLL-33, as well, if it still exists.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -8,33 +8,27 @@ const routes: Routes = [
...
-  {
-    path: 'rolodex',
-    loadChildren: './pages/rolodex/rolodex.module#RolodexPageModule'
-  },
-  {
-    path: 'contact',
-    loadChildren: './pages/contact/contact.module#ContactPageModule'
+    path:'',
+    loadChildren:'./pages/home/home.module#HomePageModule'
   },
```
Comment (brickgriff) [inline src/app/app-routing.module.ts:13]: Why delete all those lines? Also why add the redundant path to `''`, when it already exists at line 5-9?

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -8,33 +8,27 @@ const routes: Routes = [
...
   { path: 'link', loadChildren: './pages/link/link.module#LinkPageModule' },
   { path: 'faq', loadChildren: './pages/faq/faq.module#FaqPageModule' },
   { path: 'about', loadChildren: './pages/about/about.module#AboutPageModule' },
   { path: 'license', loadChildren: './pages/license/license.module#LicensePageModule' },
   { path: 'settings', loadChildren: './pages/settings/settings.module#SettingsPageModule' },
   { path: 'feedback', loadChildren: './pages/feedback/feedback.module#FeedbackPageModule' },
   { path: 'invite', loadChildren: './pages/invite/invite.module#InvitePageModule' },
-  { path: 'password', loadChildren: './pages/password/password.module#PasswordPageModule' }
+  { path: 'password', loadChildren: './pages/password/password.module#PasswordPageModule' },
+  { path: 'exchange', loadChildren: './pages/home/exchange/exchange.module#ExchangePageModule' }
```
Comment (brickgriff) [inline src/app/app-routing.module.ts:35]: Try to keep the paths shallow. I would rather we avoid nesting pages like this. Was there any reason for this?

**Comment by brandlxee11** [comment]
Right. That’s because I originally opened the issue as Roll-33 and named the branch as such until I realized that you had already opened “Roll-33: Add new field button on Profile Page”. So, I renamed the issue Roll-34, but the branch still says Roll-33 unfortunately. 😅 My bad, hope that’s not confusing. Though I think we can close the original Roll-33 issue you opened as it may not apply anymore. Please advise how we should proceed. I’ll follow your lead.

**Comment by brickgriff** [comment]
There's also supposed to be a home router module? Again, just keep the page system flat so we don't get confused. Generating a page is simple with `ionic g page pages/name-page` (generating a service looks like `ionic g service services/new-service/new-service`). We can have complex modular routing trees but is it necessary at this point?

**Review by brickgriff — COMMENTED** [review]
> Right. That’s because I originally opened the issue as Roll-33 and named the branch as such until I realized that you had already opened “Roll-33: Add new field button on Profile Page”. So, I renamed the issue Roll-34, but the branch still says Roll-33 unfortunately. 😅 My bad, hope that’s not confusing. Though I think we can close the original Roll-33 issue you opened as it may not apply anymore. Please advise how we should proceed. I’ll follow your lead.

That's fine. The full branch name is unique so it's okay.

**Comment by brickgriff** [comment]
Finally, this has to be fixed

```
] ERROR in ./src/app/app.component.scss
[ng] Module build failed (from ./node_modules/sass-loader/lib/loader.js):
[ng] @import '../theme/fonts';
```

**Comment by brandlxee11** [comment]
> There's also supposed to be a home router module? Again, just keep the page system flat so we don't get confused. Generating a page is simple with `ionic g page pages/name-page` (generating a service looks like `ionic g service services/new-service/new-service`). We can have complex modular routing trees but is it necessary at this point?

Yes, the home-routing.module.ts file was needed in order for the pages of the tabs container to route between each other properly. I think this is as complex as it's going to get given that the core dashboard of the app has been setup with this change.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (brandlxee11) [reply src/app/app-routing.module.ts:13]: Definitely not the best approach, but I needed to change things around a bit for me to actually see what I'm doing. And you're right, I noticed the redundancy after the fact, but in the next branch I actually updated the path to 'login' at line 5-9. (Remember I'm learning as I go, so my actions may come across very unorthodox at times)

File: src/app/app-routing.module.ts (hunk 2 above)
Comment (brandlxee11) [reply src/app/app-routing.module.ts:35]: Yes! This is how the tabs container on the homepage was created. The homepage holds the rolodex page, the exchange page, and the profile page (which holds the edit page). Got it?

For reference, here's the tutorial I used:
https://ovpv.me/add-tabs-ionic-4/

**Comment by brandlxee11** [comment]
> Finally, this has to be fixed
> 
> ```
> ] ERROR in ./src/app/app.component.scss
> [ng] Module build failed (from ./node_modules/sass-loader/lib/loader.js):
> [ng] @import '../theme/fonts';
> ```

Hmm, I haven't seen that error on my end at all. Must be a local issue for you.

**Pull request merged**